unit2/unit2.ipynb

```diff
@@ -566,9 +567,9 @@
       "source": [
         "def epsilon_greedy_policy(Qtable, state):\n",
         "  # Randomly generate a number between 0 and 1\n",
-        "  random_int = \n",
-        "  # if random_int > greater than epsilon --> exploitation\n",
-        "  if random_int > epsilon:\n",
+        "  random_num = \n",
+        "  # if random_num > greater than epsilon --> exploitation\n",
+        "  if random_num > epsilon:\n",
```

Comment:
It's not a showstopper but a little confusing as the variable name sounds like it would be either 0 or 1 :wink:

Reply:
Good point!